src/ProductConstructionService/ProductConstructionService.Api/PcsStartup.cs

```diff
@@ -158,6 +159,8 @@ internal static void ConfigurePcs(
         builder.Services.AddMergePolicies();
         builder.Services.Configure<SlaOptions>(builder.Configuration.GetSection(ConfigurationKeys.DependencyFlowSLAs));
 
+        await builder.Services.ConfigureRedis(builder.Configuration, isDevelopment, managedIdentityId);
```

Comment:
we don't have to pass the managedIdentity, we can get it in the configuration method

Reply:
But that would mean getting the same config key in two different places, no? One here and one in `Common`

Reply:
Nevermind, I see we are already retrieving it in other extension methods. Will fix